index.html

```diff
@@ -1651,10 +1651,11 @@ <h3>Credential Subject</h3>
           <dt><dfn class="export"
                    id="defn-credentialSubject">credentialSubject</dfn></dt>
           <dd>
-The value of the <code>credentialSubject</code> <a>property</a> is defined as
-a set of objects that MUST contain one or more claims that are each related to a
-<a>subject</a> of the <a>verifiable credential</a>. Each object MAY contain an
-<code>id</code>, as described in Section <a href="#identifiers"></a>.
+The value of the <code>credentialSubject</code> <a>property</a> is defined as a
+set of objects where each object MUST contain one or more <a>claims</a>
+expressed as property-value pairs. Each object MAY also contain an
+<code>id</code> to identify the <a>subject</a>, as described in Section
+<a href="#identifiers"></a>.
```

Comment:
Minor tweak to ensure we don't erroneously suggest a claim isn't a triple:

```suggestion
set of objects where each object MUST contain one or more <a>claims</a>
expressed as property-value pairs of the <a>subject</a>. Each object MAY
also contain an <code>id</code> to identify the <a>subject</a>, as
described in Section <a href="#identifiers"></a>.
```